tests/std/tests/P0355R7_clocks/test.cpp

```diff
@@ -0,0 +1,331 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <cassert>
+#include <chrono>
+#include <compare>
+#include <iterator>
+#include <utility>
+
+using namespace std;
+using namespace std::chrono;
+using namespace std::chrono_literals;

```

Comment:
This is unnecessary (both `using namespace std;` and `using namespace std::chrono;` make the `chrono_literals` available).